alphadia/outputtransform.py

```diff
@@ -982,6 +982,8 @@ def _build_run_stat_df(
             base_dict["rt_error"] = np.nan
             base_dict["mobility_error"] = np.nan
 
+        base_dict["gradient_length_m"] = channel_df["gradient_length"].max() / 60
```

Comment:
alternatively: `rt_max` .. and we should mb keep the unit in seconds (I guess the other rt observables rest is seconds?)

Reply:
I prefer using minutes, for example, we say 21 min gradient, 11 min gradient, but not 21*60 seconds gradient.